LayoutTest/TestCase/LYTLayoutFailingTestSnapshotRecorder.h

```diff
@@ -0,0 +1,27 @@
+// © 2016 LinkedIn Corp. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+#import <XCTest/XCTest.h>
+
+@interface LYTLayoutFailingTestSnapshotRecorder : NSObject<XCTestObservation>
+
+@property (nonatomic, strong) UIView *viewUnderTest;
```

Comment:
Yeh that makes sense and feels cleaner passing them in with the `saveImageOfCurrentView` method. I'll make the change and have LYTLayoutTestCase hold onto the current view and data.